### DISCUSSION TITLE
/printing-temp-tower-to-get-better-3d-prints/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#7] [Announcements]
Printing a Temp Tower to get better quality 3D prints
        
[https://blog.ktz.me/printing-temp-tower-to-get-better-3d-prints/](https://blog.ktz.me/printing-temp-tower-to-get-better-3d-prints/)

**Comment by kyleinprogress** [comment]
For anyone that finds this and wants a more "GUI Friendly" way, check out this youtube video. You can add custom gcode per layer in Prusa Slicer.

https://www.youtube.com/watch?v=NEnVQKX-_N8
